android/tangram/src/com/mapzen/tangram/DebugFlags.java

```diff
@@ -0,0 +1,57 @@
+/* ----------------------------------------------------------------------------
+ * This file was automatically generated by SWIG (http://www.swig.org).
+ * Version 3.0.7
+ *
+ * Do not make changes to this file unless you know what you are doing--modify
+ * the SWIG interface file instead.
+ * ----------------------------------------------------------------------------- */
+
+package com.mapzen.tangram;
+
+public final class DebugFlags {
+    public final static DebugFlags freeze_tiles = new DebugFlags("freeze_tiles", 0);
```

Comment:
Oh ha I see now that this file is generated. In that case all codestyle bets are off ;)